inst/tests/tests.Rraw

```diff
@@ -21062,3 +21062,16 @@ capture_error_message <- function(expr) {
   msg
 }
 test(2306, {msg <- capture_error_message(uniq_diff(as.integer(c(1, 2, -1, 4)), 4, FALSE))print(msg)return(grepl("\\[-1\\]", msg))}, TRUE)
+
+# test for enhancing error message of invalid column #6512
+uniq_diff <- function(...) {
```

Comment:
I don't think these helper functions (uniq_diff and capture_error_message) should be necessary, please remove